Tests/DUnitX.Tests.Example.pas

```diff
@@ -141,7 +158,7 @@   TExampleFixture3 = class
 implementation
 
 uses
-  DUnitX.DUnitCompatibility;
+  DUnitX.DUnitCompatibility, System.DateUtils;
```

Comment:
Needs  {$IFDEF USE_NS} like the other units to ensure compatibility with D2010/XE.